.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -3,13 +3,18 @@ Its much easier for us to read and review a PR that is small, linted, and contai
 
 ## Description
 
-_Describe the purpose of and changes within this Pull Request._
+This pull request improves several aspects of the verify process. 
+- Retries any (known) network related issues during the fetches.
+- Outputs cleaned log values without nulls.
+- Presents all of the issues at the end, with some recommendations.
+- Does not replace links with null, and better handles on-chain true/false state.
+- Stores a hash of the link when checked out. Will not re-check on future runs if the link doesn't change.
 
 ## References
 
-_List links to any related tasks (GitHub Issues, Pull Requests, 3rd part task managers)_
+https://github.com/metaplex-foundation/metaplex/issues/1030
 
 ## Testing
 
-_How was this tested? If manually tested, list all steps necessary for the reviewer to confirm._
+Testing involves just running verify. If working with a large set of assets, the network issues seem frequent.  If not, changing the link uri in the cache to invalid uris will test some of the situations this handles.
```

Comment:
Hey - this looks interesting, but you don't need to edit this file - it's the template used for all pull requests to the repo.

Reply:
Sorry, I am very new to PRs. I thought editing it was how I use it in my PR.  Am I supposed to do anything with it?

Reply:
Or would it have prompted me to answer those questions if I submitted my PR after it existed?

Reply:
Hey no need to apologise mate, this PR is great :)

This content would get prefilled into the PR description when you opened it on the MP repo. Safe to just remove these changes from your branch :)